src/standalone.js

```diff
@@ -21,7 +21,7 @@ export default function ({
   syntax,
   formatter = "json",
 } = {}) {
-  if (!files && !code || files && code) {
+  if (!files && !code && code !== "" || files && (code || code === "")) {
```

Comment:
Could this be simplified to

```js
var codeWasPassed = typeof code === "string"
if (!files && !codeWasPassed || files && codeWasPassed)
```

That's untested, just a suggestion. What do you think? I think it might make it easier to understand why the logic is there when coming back to this several months from now.